main.rb

```diff
@@ -0,0 +1,98 @@
+require_relative './person'
+require_relative './book'
+require_relative './app'
+
+def selected_option_number
+  options = [
+    'list all books',
+    'list all people',
+    'create a person',
+    'create a book',
+    'create a rental',
+    'list all rentals for a give person id',
+    'exit'
+  ]
+  puts
+  puts 'Please choose an option by entering a number: '
+  options.each_with_index do |option, index|
+    puts "#{index + 1} - #{option.capitalize}"
+  end
+  gets.chomp.to_i
+end
+
+def add_person(app)
```

Comment:
There is an error while creating a person. Kindly see the errors below and make the user is able to create a person.

![person](https://user-images.githubusercontent.com/35504149/199793442-cb547925-416f-44b2-971b-debfc8e68b2c.png)


![person2](https://user-images.githubusercontent.com/35504149/199793466-c5c579f9-a6cc-4462-a5ed-bc9d84e915d8.png)